docs/pages/deployment/cli-reference.rst

```diff
@@ -422,7 +422,7 @@ Registers a new DID
       --capabilityInvocation   Pass 'false' to disable capabilityInvocation capabilities. (default true)
       --controllers strings    Comma-separated list of DIDs that can control the generated DID Document.
   -h, --help                   help for create-did
-      --keyAgreement           Pass 'true' to enable keyAgreement capabilities.
+      --keyAgreement           Pass 'false' to disable keyAgreement capabilities. (default true)
```

Comment:
I find these sentences hard to read, because they're inverted (applies to the other verification method flags as well). I'd rather read something like: "Whether to register a keyAgreement key on the DID. (default true)".

Reply:
I agree on hard to read, but it does provide a consistent indication of the default value. The generated code now contains the default value as `(default true)` only when the value deviates from the default value for the type in go. So when bool=false the default value is missing from the docs which I find confusing.